src/main/java/com/google/sps/data/ScheduledCourse.java

```diff
@@ -83,7 +83,7 @@ public int hashCode() {
     return super.hashCode() * Objects.hash(lectureSection, labSection);
   }
 
-  public Section getLectureSection() {
+  public Section getlectureSection() {
```

Comment:
It looks like this typo has since been fixed in the HandleRecitations branch (I'm guessing heidi also noticed it and I'm assuming fixed it in her PR). This means that you can actually rename it to the proper thing (`getLectureSection`), refactor the rest of your code accordingly, and it should work!

Reply:
Ok sounds good. I will change it to `getLectureSection()`